charts/oes/values.yaml

```diff
@@ -283,7 +283,7 @@ dashboard:
   image:
     registry: quay.io/opsmxpublic
     repository: ubi8-oes-dashboard
-    tag: v3.8.0
+    tag: v3.8
```

Comment:
why did we change this? it should be v3.8.x

Reply:
fixed it